app/controllers/application_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:username, :email, :password) }
```

Comment:
Metrics/LineLength: Line is too long. [91/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)